hail/python/hailtop/hailctl/__main__.py

```diff
@@ -5,7 +5,7 @@
 import re
 import time
 
-import hailctl
+from .. import hailctl
```

Comment:
er, I think this is wrong...

Reply:
This should be `from hailtop import hailctl`. Relative imports shouldn't work here

Reply:
as I understand it, when __main__ is run as a standalone script, it can't also be a module -- so this relative import doesn't work.

Reply:
Having tested this. It does work. But I'll change it.

Reply:
huh. Maybe there are some circumstances where it does, and doesn't.